troposphere/secretsmanager.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015, Mark Peek <mark@peek.org>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSObject, AWSProperty, Tags
+from .validators import (integer,
```

Comment:
The parentheses and comma could be removed since it all fits on a single line.

Reply:
done, thanks